include/seqan3/core/simd/simd_algorithm.hpp

```diff
@@ -275,20 +275,28 @@ constexpr target_simd_t upcast_unsigned(source_simd_t const & src)
  * dst[63:0] := src[i+63:i]
  * ```
  */
+template <uint8_t index, simd::simd_concept simd_t>
+constexpr simd_t extract_halve(simd_t const & src)
+{
+    static_assert(index < 2, "The index must be in the range of [0, 1]");
+
+    return detail::extract_impl<2>(src, index);
+}
+
 template <uint8_t index, simd::simd_concept simd_t>
```

Comment:
documentation missing

Reply:
I \cond out all native simd overloads, because they can't be documented (they have all the same signature for doxygen since doxygen doesn't see the requires)